src/burnchains/burnchain.rs

```diff
@@ -1213,12 +1213,21 @@ impl Burnchain {
         );
```

Comment:
I think the debug statement above should be 
```
 debug!(
            "Sync'ed headers from {} to {}. DB at {}",
            sync_height, end_block, db_height
         );
```

Reply:
Addressed with latest